src/main/java/io/citrine/jcc/search/pif/result/PifSearchHit.java

```diff
@@ -298,4 +372,7 @@ public Object getExtractedValueOrDefault(final String key, final Object defaultV
 
     /** Map of extracted fields. */
     private Map<String, Object> extracted = new HashMap<>();
+
+    /** Map of paths to extracted fields. */
+    private Map<String, String> extractedPath = new HashMap<>();
```

Comment:
Can the path be something more useful/structured than a string?

Reply:
To summarize what we talked about in person -- This is a universal way to return the information that can be parsed by any client library. We can add additional functionality in the future to use the extracted path to operate on the PIF system that is returned.